### PR TITLE
FE browser tests

### PR DESCRIPTION
**Pull request by fna**
1. Don't test pagination
2. Highlights tests randomly fail/succeed, don't know what's up

### CODE REVIEW

**Comment by amymok** [comment]
@fna Push your loan_estimate.py file up as well, I am getting an import error.

File: test/browser_testing/features/closing_disclosure.feature [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+@closing_disclosure
+Scenario Outline: Test that resizing window size changes image size too
+  When I click the tab "<tab_name>"
+    When I resize window image "<css_selector>" size changes too
+
+Examples:
+  | tab_name    | css_selector          |
+  | Checklist   | div.illustration img  |
+  | Definitions | img.image-map_image   |
+
```
Comment (virginiacc) [inline test/browser_testing/features/closing_disclosure.feature]: I think these should both be the same selector -- `img.image-map_image`

File: test/browser_testing/features/closing_disclosure.feature [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+@closing_disclosure
+Scenario Outline: Test overlays/highlights
+  When I click the tab "<tab_name>"
+    When I hover over an overlay the corresponding explainer has class has-attention
+
+Examples:
+  | tab_name    |
+  | Checklist   |
+  | Definitions |
```
Comment (virginiacc) [inline test/browser_testing/features/closing_disclosure.feature]: Every time the overlay test runs, I'm getting an error for just the first item in the Checklist tab -- `#page-1-checklist-check-name`. It's failing the `has-attention` class check. If the order is affecting the results, I wonder if doing a second check for the failed items would work?

Comment (fna) [inline test/browser_testing/features/closing_disclosure.feature]: I'll see what I can do. Don't merge yet

File: test/browser_testing/features/closing_disclosure.feature (hunk 1 above)
Comment (fna) [inline test/browser_testing/features/closing_disclosure.feature]: Yes, both has to be the same, but I think more specific like div.illustration img.image-map_image

**Comment by virginiacc** [comment]
@fna Looks great -- everything's passing for me now, so I think this is ready to merge! :+1:

**Pull request merged**